src/test/java/de/rabea/ui/UserInterfaceTest.java

```diff
@@ -102,7 +102,7 @@ public void returnsHvHGameModeWhenUserEnters1() {
     public void asksTheUserAgainForGameModeIfBadInputIsEntered() {
         fakeConsole.userInput("hello", "1");
         userInterface.chooseGameMode();
-        assertEquals("Please enter 1 if you want to play against the computer and 2 if you want to play against another human player.\n", fakeConsole.messagePrinted());
```

Comment:
I want one and only one test to fail if we have to change a detail of the user interface. Do we need both of these tests? Or can we DRY it up so only one test fails the next time we make a UI change?